### ISSUE TITLE
O2 tanks not refilling to full (2533 kPa)

### ISSUE THREAD

**Issue by PVT-LF** [tgstation/tgstation#76639]
## Reproduction:

O2 tanks can't be refilled to 2533 kPa. Will conduct more testing & locate problem PR later as the round has not ended yet.

EDIT: upon testing on a current build, there was no problem.

I latejoined a event round, nations gamemode, Cargo Hub map so it could be that.

**Comment by TheVekter** [comment]
Let me know if you can reproduce this again reliably, otherwise I'm letting it close in a week.

**Comment by PVT-LF** [comment]
> Let me know if you can reproduce this again reliably, otherwise I'm letting it close in a week.

At the moment it seems to be isolated to what was an event round on an event map, it hasn't occurred again in normal gameplay or tests

**Comment by Sealed101** [comment]
how exactly could they not be refilled? did they not reach the expected pressure? did they not fill up at all? what was the "donating" canister/tank's mol count and pressure? was the tank that you've tried to refill damaged?

**Comment by TheVekter** [comment]
> > Let me know if you can reproduce this again reliably, otherwise I'm letting it close in a week.
> 
> At the moment it seems to be isolated to what was an event round on an event map, it hasn't occurred again in normal gameplay or tests

Right, but neither of those factors should be causing what you're describing, so if you notice it happen again sometime before this issue closes, let us know so I can remove the Cleanup Flagged label.

**Comment by PVT-LF** [comment]
> how exactly could they not be refilled? did they not reach the expected pressure? did they not fill up at all? what was the "donating" canister/tank's mol count and pressure? was the tank that you've tried to refill damaged?

The tank I used was that from the survival box and another from a tank dispenser. The pressure did not increase when I opened the valve after setting the pressure regulator to max.

The donating canister was approx. full and I don't believe anyone messed with it as it appeared to be the default spawn canister for topping up O2.

I was a latejoin miner that round so I didn't have any gas analyzers to check for mol count etc and the event round ended around 10 mins after I joined so unfortunately I didn't get to test much and I had initially assumed it would be reproducible during normal rounds.

Anyway it's pretty minor so thanks for helping to look into it.
